themes/bootstrap5/templates/feedback/response.phtml

```diff
@@ -1,15 +1,10 @@
 <?php
   $form = $this->form;
   $form->prepare();
-
-  $title = $form->getTitle();
-  $title = !empty($title) ? $this->translate($title) : null;
 ?>
 <?php if (!$this->inLightbox): ?><div class="feedback-content"><?php endif; ?>
-<?php if ($title): ?>
-  <?php $this->headTitle($title); ?>
-  <?php $headTag = $this->inLightbox ? 'h2' : 'h1'; ?>
-  <<?=$headTag?>><?=$this->escapeHtml($title)?></<?=$headTag?>>
+<?php if ($title = $form->getTitle()): ?>
```

Comment:
Same here: is it concerning that we will no longer have an h1?

Reply:
Adjusted to have level 1 and in component to check if is in lightbox and level is 1, changes it to 2